tools/run_tests/build_package_node.sh

```diff
@@ -58,6 +58,8 @@ tools_version=$(npm list | grep -oP '(?<=grpc-tools@)\S+')
 output_dir=$artifacts/grpc-precompiled-binaries/node/grpc-tools/v$tools_version
 mkdir -p $output_dir
 
+well_known_protos=( any api compiler/plugin descriptor duration empty field_mask source_context struct timestamp type wrappers )
```

Comment:
compiler/plugin protos is probably unneeded here (but you can have it if you want to).
Here's list of .proto files for Protobuf.Tools nuget.
https://github.com/google/protobuf/blob/master/csharp/Google.Protobuf.Tools.nuspec